pkg/tarmak/vault/vault.go

```diff
@@ -24,6 +24,7 @@ const (
 	VaultStateUnsealed
 	VaultStateUnintialised
 	VaultStateErr
+	vaultTunnelCreationTimeoutSeconds = 5
```

Comment:
Please test multi/single cluster E2E, I am not too sure if that is a good value

Reply:
It worked fine with 5, but have upped it to 10 to be safe